packages/ui/src/theme/utils.ts

```diff
@@ -48,3 +60,14 @@ interface NameTransformProps {
 export function cssNameTransform({ path = [] }: NameTransformProps): string {
   return `${kebabCase([CSS_VARIABLE_PREFIX, ...path].join(' '))}`;
 }
+
+/**
+ * Helper function to test if something is a design token or not.
+ * Used in the React component style props.
+ *
+ * @param value - thing to test if it is a design token or not
+ * @returns boolean
+ */
+export function isDesignToken(value: unknown): value is WebDesignToken {
+  return isObject(value) && has(value, 'value');
```

Comment:
Is there any advantage to using lodash util vs just `Object.hasOwn(value, 'value')`. Also might be worth considering renaming the value prop. The check for whether a value has property value is less intuitive. Maybe `object` or `token`?

Reply:
Is `Object.hasOwn` available for use with `ESNext`?

Reply:
I believe so, it has been added to the Object standard API a while back (2017?) and is supported by all major browsers: https://caniuse.com/?search=Object.hasOwn